arxiv/avoiding_catastrophe/active_dendrites_enable_multitask_learning/README.md

```diff
@@ -0,0 +1,28 @@
+# Project: multitask reinforcement learning in MetaWorld using dendritic networks.
+
+
+Experiments with multitask reinforcement learning, using dendritic networks, tested in MetaWorld.
+
+For more details please contact lsouza at numenta dot com.
+
+### Installation
+
+Important: MetaWorld requires Mujoco. For information about installing and acquiring a license, please refer to the [Mujoco github repo](https://github.com/openai/mujoco-py)
```

Comment:
Followed the instructions to install `MuJoCo` from the github repo and received the following error
```
You appear to be missing MuJoCo. ....
```
Is there a specific version of MuJoCo I need to install?

Reply:
It should work with any version afaik. I see they recently updated the version listed in their Readme, but metaworld doesn't say anything about that, they list the same instructions we do: `"Please follow the instructions in the mujoco-py package for help."`

Reply:
I've followed this https://github.com/openai/mujoco-py#install-mujoco for OSX and it installed `~/mujoco/mujoco210`. When I ran `conda env create` it failed with the following:
```
You appear to be missing MuJoCo. We expected to find the file here: ~/.mujoco/mujoco200
```